README.md

```diff
@@ -151,12 +151,12 @@ Or set the `GOOGLE_CLOUD_PROJECT` environment variable to your gcp project id.
 
 ## Frontend Tests
 
-The frontend tests use NodeJS to run. You will need to have node version 12.X to run these tests. To do this, do the following commands:
+The frontend tests use NodeJS to run. You will need to have node version 14.X to run these tests. To do this, do the following commands:
 
 ``` shell
 brew install nvm
-nvm install 12
-nvm use 12
+nvm install 14
+nvm use
```

Comment:
```suggestion
nvm use 14
```

Reply:
Because I updated the `.nvmrc` file with the correct version of node, you no longer have to specify the version you want to use when running `nvm use`. It will now always get the correct version to be used for the repo. Because of this there is less chance of devs running the tests with the wrong version of node.

Reply:
In fact, I've just realised we no longer have to specify it when we run `nvm install` so will remove it from there too